src/dmd/iasmdmd.d

```diff
@@ -817,24 +817,25 @@ RETRY:
         }
         case 2:
         {
-            //printf("opflags1 = "); asm_output_flags(opflags[0]); printf(" ");
-            //printf("opflags2 = "); asm_output_flags(opflags[1]); printf("\n");
+            enum log = false;
+            if (log) { printf("opflags1 = "); asm_output_flags(opflags[0]); printf("\n"); }
```

Comment:
`static if` and the next 6 times too.

Reply:
In this case, `if` does the same thing and is shorter, too, as the line is getting a bit long. The code for the printf isn't generated.